feat: use key file to encrypt (#19)
Kong/lua-resty-lmdb: README.md

## Directives

### lmdb_encryption_key

**syntax:** *lmdb_encryption_key path/to/keyfile;*

**context:** *main*

Encrypt the lmdb database. Encryption is enabled only when the `lmdb_encryption_key` is set. The

> content of keyfile will be used as the key to encrypt lmdb.

Reviewer: issue from previous PR though
```suggestion
content of keyfile will be used to derive a key to encrypt lmdb.
```

Author: done.
